book/src/customizing-ci.md

```diff
@@ -0,0 +1,63 @@
+# Customizing CI
+
+Out of the box, cargo-dist's CI configuration is designed to cover the majority of usecases. We provide some tools to customize how it works for your project.
+
+## Build and upload artifacts on every pull request
+
+> since 0.3.0
+
+By default, cargo-dist will run the plan step on every pull request but won't perform a full release build. If these builds are turned on, the resulting pull request artifacts won't be uploaded to a release but will be available as a download from within the CI job. To enable this, select the "upload" option from the "check your release process in pull requests" question in `cargo-dist-init` or set the `pr-run-mode` key to `"upload"` in `Cargo.toml`. For example:
```

Comment:
might wanna say `[workspace.metadata.dist]` or "cargo-dist config" to be a bit less ambiguous (not a top level Cargo key)?

Reply:
Good call.